### PR TITLE
Fixup dead links

### PR DESCRIPTION
**Pull request by evantahler**
Fixup dead links that @sophia-wiley found

```
File: ./airbyte-integrations/bases/standard-source-test/readme.md
Link: ../base-python-test/readme.md
Link: ../base-python-test/base_python_test/test_iface.py
Status: both 400

File: ./tools/ci_connector_ops/ci_connector_ops/pipelines/README.md
Link: https://www.youtube.com/watch&ab_channel=Dagger
Status: 404
```

### CODE REVIEW

File: airbyte-integrations/bases/standard-source-test/readme.md [hunk 1]
```diff
@@ -3,31 +3,35 @@
...
 We will _not_ merge sources that cannot pass these tests unless the author can provide a very good reason™.
 
 ## What tests do the standard tests include?
+
 Check out each function in [SourceAcceptanceTest](src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java). Each function annotated with `@Test` is a single test. Each of these functions is proceeded by comments that document what they are testing.
 
 ## How to run them from your integration?
-* If writing a source in Python, check out this [readme](../base-python-test/readme.md)
+
+- If writing a source in Python, **don't use this.**
```
Comment (evantahler) [inline airbyte-integrations/bases/standard-source-test/readme.md:27]: 🐍

File: tools/ci_connector_ops/ci_connector_ops/pipelines/README.md [hunk 2]
```diff
@@ -113,37 +122,37 @@ Please ignore the `Set your environment variables` step and focus on running `co
...
-I'll write test once the code architecture is ratified.
+   Dagger team answer: Get inspirations from their own repo [here](https://github.com/dagger/dagger/tree/main/sdk/python/tests).
+   I'll write test once the code architecture is ratified.
 
 7. What would be your suggestion to run `docker scan` from dagger to spot vulnerabilities on our connectors?
-Dagger team answer: A scanning tool should be wrapped in a container to scan images from Dagger.
+   Dagger team answer: A scanning tool should be wrapped in a container to scan images from Dagger.
 8. Do you have a tool to re-order logs line by order of pipeline after execution?
-A log grouping tool is under construction: https://www.youtube.com/watch&ab_channel=Dagger
+   A log grouping tool is under construction
```
Comment (evantahler) [inline tools/ci_connector_ops/ci_connector_ops/pipelines/README.md:139]: @alafanechere I don't know what you meant to link, but I don't think it matters to much here 🤷

File: airbyte-integrations/bases/standard-source-test/readme.md (hunk 1 above)
Comment (bnchrch) [reply airbyte-integrations/bases/standard-source-test/readme.md:27]: Wait... why have this line? is there a place we can link them too?

**Review by bnchrch — COMMENTED** [review]
I feel a bit confused. Should we not tell users how to run their integration tests?

**Review by sophia-wiley — APPROVED** [review]

**Comment by evantahler** [comment]
Oops... automerged!

> Wait... why have this line? is there a place we can link them too?

So this package is a special collection of /unit tests/ that only the java database sources use.  python connector do their own thing now.  Everyone uses CAT, and that's a different package.

**Comment by bnchrch** [comment]
ah ok that makes sense!

**Pull request merged**